src/axom/mir/MIRAlgorithm.cpp

```diff
@@ -132,5 +132,17 @@ void MIRAlgorithm::saveMesh(const conduit::Node &n_mesh, const std::string &file
 #endif
 }
 
+std::string MIRAlgorithm::localPath(const conduit::Node &obj) const
+{
+  std::string path(obj.path());
+  const auto dpos = path.find("dom");
+  const auto spos = path.find("/");
+  if(dpos == 0 && spos > dpos && obj.parent() != nullptr)
+  {
+    path = path.substr(spos + 1, path.size() - spos - 1);
+  }
+  return path;
```

Comment:
Could this be replaced by 
```cpp
return axom::utilities::string::removePrefix("dom", obj.path());
```
https://github.com/LLNL/axom/blob/0398fe6f3191ff6e71aa7baf8764e34a8efc10e3/src/axom/core/utilities/StringUtilities.hpp#L136-L150

Reply:
Thanks -- that makes sense.